docs/DeveloperGuide.md

```diff
@@ -767,21 +767,41 @@ The following sequence diagrams shows how the unmark command works:
 
 #### Description
 
-The `clear` command deletes all students currently stored in TAPA. During the execution of the `clear` command, a new `ClearCommand` object will be created, and is executed by the `LogicManager`. The `MainWindow` changes the `commandBox` to only recognise a followup `confirm` command. If the user executes a `confirm` command, a new `ConfirmClearCommand` object is created and executed by the `LogicManager`. Subsequently, `ModelManager` clears the students currently stored in TAPA.
+The `clear` command deletes all students currently stored in TAPA. Before all the students are deleted, the user will have to confirm their decision by inputting `confirm`.
 
 #### Implementation
-1. Upon receiving the user input, the `LogicManager` starts to parse the given input text using `AddressBookParser#parseCommand()`.
+1. When the user inputs `clear`, the `LogicManager` parses the given input text using `AddressBookParser#parseCommand()`.
```

Comment:
```suggestion
1. When the user inputs "clear", the `LogicManager` parses the given input text using `AddressBookParser#parseCommand()`.
```

I think for the rest of the DG, we refer to user inputs in " quotes instead of ` quotes, so maybe it would be better to standardise it this way?